### PR TITLE
1.1.3.3_Difference_Array.md

### LINKED ISSUE #409: DSA 1.1.3.3 Arrays: Range Queries
## Description
What is a difference array and how is it used to perform range update queries?

## Task
- [ ] Documentation
- [ ] Audio
- [ ] Video

## Note
- While requesting the issue to be assigned, please mention your **NAME & BATCH NUMBER**.
- Kindly refer to [Issue Guidelines](https://github.com/girlscript/winter-of-contributing/blob/DSA/DSA/CONTRIBUTING.md#issues).
- In case you see that the issue has already been assigned, try to request for a different task within the same issue.
- Changes should be made inside the ````DSA/directory```` & ````DSA```` branch.
- Name of the file should be same as the issue.
- This issue is only for 'GWOC' contributors of 'DSA' Domain.

### PR DESCRIPTION
**Pull request by Ayushi2811**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #409 

<hr>

## Type of change 📝:Added the required Documentation

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [x] DSA
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Ayushi2811** [comment]
@nimisha-yadav Completed the documentation and commited the the PR for issue#409.
Kindly review it.

**Review by Sin-Sumit — APPROVED** [review]

**Comment by nimisha-yadav** [comment]
Please review @sneha-nitdgp @harshsri28

File: DSA/1.1 Arrays/3. Range Queries/DSA 1.1.3.3 Arrays: Range Queries.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Difference Arrays
+
+As we can grab the definition from the name itself “difference array” which says this is a type of array which contains differnece of something.
+
+
+So the next question arises ,whose difference?
+
+
+The answer to this is that it consist the difference of the adjacent element present in the original array.So difference array is the array which contains the difference of the adjacentelements of the former array to which it belongs or from where it is extracted
```
Comment (harshsri28) [inline DSA/1.1 Arrays/3. Range Queries/DSA 1.1.3.3 Arrays: Range Queries.md:9]: fix typo i.e adjacentelements

**Review by harshsri28 — CHANGES_REQUESTED** [review]

File: DSA/1.1 Arrays/3. Range Queries/DSA 1.1.3.3 Arrays: Range Queries.md (hunk 1 above)
Comment (Ayushi2811) [reply DSA/1.1 Arrays/3. Range Queries/DSA 1.1.3.3 Arrays: Range Queries.md]: Done these changes @harshsri28

**Review by Ayushi2811 — COMMENTED** [review]
done changes @harshsri28

**Review by harshsri28 — APPROVED** [review]
LGTM

**Comment by Ayushi2811** [comment]
when will this issue get closed so that i can get other to be assigned?
@harshsri28

**Comment by harshsri28** [comment]
supervisor has a right to merge.

**Review by sneha-nitdgp — APPROVED** [review]

**Review by nimisha-yadav — COMMENTED** [review]
LGTM. Just rename your file 1.1.3.3_Difference_Array.md

**Comment by Ayushi2811** [comment]
@nimisha-yadav  changes done

**Comment by nimisha-yadav** [comment]
> @nimisha-yadav changes done

You changed the name of the PR and not the file. Rename your file @Ayushi2811

**Comment by Muskan-creator** [comment]
Congrats on getting your pr merged🎉.Thanks for contributing

**Pull request merged**